sdk/identity/azure-identity/src/test/java/com/azure/identity/ManagedIdentityCredentialTest.java

```diff
@@ -105,11 +105,24 @@ public void testArcUserAssigned() {
     public void testInvalidIdCombination() {
         // setup
         String resourceId = "/subscriptions/" + UUID.randomUUID() + "/resourcegroups/aresourcegroup/providers/Microsoft.ManagedIdentity/userAssignedIdentities/ident";
+        String objectId = "2323-sd2323s-32323-32334-34343";
 
         // test
         Assertions.assertThrows(IllegalStateException.class,
             () -> new ManagedIdentityCredentialBuilder()
                 .clientId(CLIENT_ID).resourceId(resourceId).build());
+
+        Assertions.assertThrows(IllegalStateException.class,
+            () -> new ManagedIdentityCredentialBuilder()
+                .clientId(CLIENT_ID).resourceId(resourceId).objectId(objectId).build());
```

Comment:
do we need this assert? I think the pair testing of the combos with the other assertions covers it?